src/components/InventoryDetail/TopBar.js

```diff
@@ -46,19 +46,18 @@ const TopBar = ({
   showTags,
 }) => {
   const dispatch = useDispatch();
-  const navigate = useInsightsNavigate();
+  const navigate = useInsightsNavigate('inventory');
```

Comment:
Maybe it is not worth it, but just an idea: can we use the appName from the package.json file? That would maybe help to have a consistent app name everywhere.